lib/pages/widget_list.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+
+const Color WHITE = Colors.white;
+const Color BLACK = Colors.black;
+const Color GREY = Colors.grey;
+
+class WidgetLists extends StatefulWidget {
+  final String _listUuid;
+  final String _itemPicture;
+  final String _numberOfItemShared;
+
+  WidgetLists(this._listUuid, this._itemPicture, this._numberOfItemShared);
+
+  State<StatefulWidget> createState() {
+    return WidgetListsState(_listUuid, _itemPicture, _numberOfItemShared);
+  }
+}
+
+class WidgetListsState extends State<WidgetLists> {
+  final String _listUuid;
+  final String _itemPicture;
+  final String _numberOfItemShared;
+  WidgetListsState(this._listUuid, this._itemPicture, this._numberOfItemShared);
+
+  String label = "";
+  String count = "";
+
+  Future<void> getListDetails() async {
+    String labelValue;
+    String countValue;
+
+    await Firestore.instance
```

Comment:
group the two Firebase call in one

await Firestore.instance
        .collection("wishlists")
        .document(_listUuid)
        .get()
        .then((value) { 
            labelValue = value["label"];
            countValue = value["itemCounts"];
});